### DISCUSSION TITLE
change protobuf specification ( file name )

### DISCUSSION THREAD

**Discussion by gosunuts** [aergoio/aergo#257] [Show and tell]
To update the Protobuf and gRPC versions, there must not be the same proto file name.
However, the filename "raft.proto" is duplicated within the etcd repository.
Therefore, I renamed the "raft.go" file to "aergo_raft.go." You can find the updated files at this link: https://github.com/aergoio/aergo-protobuf/tree/topic/update-dependency.
It appears that changing the file name will not result in any unexpected issues.
But, after it is merged into the main branch, all repositories using Protobuf should be updated accordingly.

**Comment by kroggen** [comment]
OK, no problem

Is there a sync test running for the changes on gRPC? Is the update of libp2p included on sync test?

Will it solve all the [4 issues](https://github.com/aergoio/aergo/security/dependabot) from dependabot?

**Reply by gosunuts** [reply]
sync test finished successfully.

**Reply by gosunuts** [reply]
libp2p update is not included until now. it update only about grpc and protobuf, so dependabot alerts about libp2p can solve next time.

**Reply by gosunuts** [reply]
libp2p have many breaking change, so updating libp2p need many tests.

**Reply by kroggen** [reply]
OK, it can be done later
